drud/ddev
# Web container maintenance, fixes #1350, fixes #920, fixes #1337

## The Problem/Issue/Bug:

We had some trivial web container maintenance in various issues.

* #1350 wants memcached added for php7.3, it's ready now
* #920 requests ghostscript as it's important for specific TYPO3 needs
* #1337 requests that the ~/.ssh directory have ownership that matches the running user.
* Add php-apcu package per [stack overflow answer](https://stackoverflow.com/a/54049172/215713), and add @mglaman's Drupal 8 settings.ddev.php improvement with `$settings['class_loader_auto_detect'] = FALSE;`

## Manual Testing Instructions:

* use the Circleci build or `webimage: drud/ddev-webserver:20190107_web_container_maint` in config.yaml and `ddev start`
* Verify that php7.3-memcached and apcu are in the container (php -i, admin/reports/status/php)
* Verify that ~/.ssh is owned by the container user.
* Run ghostscript (gs)

## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:
